page-objects/pages/teamsPages/TMSTeamMembersPage.js

```diff
@@ -30,10 +30,10 @@ module.exports = {
 	addTwoTeamMemebers: async function (teammember1, teammember2) {
 		await this.addTeamMembersSteps(teammember1);
 		await this.addTeamMembersSteps(teammember2);
-		await this.submitAddTeammemberAfterAllMemebersWereAdded();
+		await this.submitAddTeamMemberAfterAllMemebersWereAdded();
 		await driver.pause(1500);
 	},
-	submitAddTeammemberAfterAllMemebersWereAdded: async function () {
+	submitAddTeamMemberAfterAllMemebersWereAdded: async function () {
```

Comment:
```suggestion
	clickSubmitAddTeamMember: async function () {
```
AfterAllMemebersWereAdded -> it is not needed
its click method

Reply:
done